### PR TITLE
Add page clarification to 3/within ; closes #735

### LINKED ISSUE #735: Prezi and Search are inconsistent re within
Presentation says that:
> Each annotation list may be part of one or more layers.

Search says that:
> All of the pages are within a layer that represents the entire resultset of matched annotations.

Thus Prezi is 0..n, and Search is 1..1.  An AnnotationList as the result of a search cannot be within multiple Layers, and any ordering asserted about Lists would make a mess of Prezi's 0..n.

Propose dropping Prezi to 0..1

### PR DESCRIPTION
**Pull request by azaroth42**
Link: http://within735.iiif.io/api/presentation/2.1/#within

### CODE REVIEW

**Comment by jpstroop** [comment]
:+1:

File: source/api/presentation/2.1/index.md [hunk 1]
```diff
@@ -320,7 +320,8 @@ A link to a machine readable document that semantically describes the resource w
 ##### within
 A link to a resource that contains the current resource, such as annotation lists within a layer. This also allows linking upwards to collections that allow browsing of the digitized objects available.
 
- * Any resource type _MAY_ be within one or more containing resources.
+ * Collections or annotation lists that serve as [pages][paging] _MUST_ be within exactly one paged resource.
+ * Other resource types, and collections or annotation lists not serving as pages, _MAY_ be within one or more containing resources.
```
Comment (jpstroop) [inline source/api/presentation/2.1/index.md]: I'd prefer:

> Other resource types, including collections or annotation lists not serving as pages, _MAY_ be within one or more containing resources.

collections or annotation lists are still resources, "and" implies something else....but it's a tiny thing.

**Comment by mikeapp** [comment]
:+1:

**Comment by zimeon** [comment]
:+1: with @jpstroop's change

**Comment by tomcrane** [comment]
:+1:

**Pull request merged**